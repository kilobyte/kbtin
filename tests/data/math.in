#ali m {#math x {$1};#strcmp {$x} {$2} {#nop} #else {#show {Error: $1 => $x expected $2}}}
m -0.25 -0.25
m -1.25 -1.25
m 2+2 4
m 2.1+2.2 4.3
m 2*2   4
m 2*-2 -4
m -2*2 -4
m -2*-2 4
m 10*0.1 1
m 2/2 1
m 2/1 2
m 2//2 1
m 2//1 2
m 10//3 3.3333333333
m 0.999999999999999999999 1
m 1//(1//3) 3
m -1//(-1//-3) -3
m T F
m 5%3 2
m -5%3 -2
m 5%-3 2
m 4.6%2.1 0.4
