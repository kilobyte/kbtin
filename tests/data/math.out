Error: T => 1 expected F
